mmv1/provider/file_template.rb

```diff
@@ -80,11 +80,7 @@ def generate(pwd, template, path, provider)
     # template is used to determine metadata about the file based on how it is
     # generated
     def format_output_file(path, template)
-      if path.end_with?('.py') && @env[:pyformat_enabled]
-        run_formatter("python3 -m black --line-length 160 -S #{path}")
-      elsif path.end_with?('.go') && @env[:goformat_enabled]
-        run_formatter("gofmt -w -s #{path}")
```

Comment:
`gofmt` runs in `-s` mode which `goimports` doesn't support, and `gofmt` runs on all go files while `goimports` doesn't run on third_party files. We'll want to keep this one!

Reply:
ah, I see.. I wonder if it would be an improvement if we run gofmt at the end instead of iteratively every time.

Reply:
I've made the modification to add back gofmt